web/src/server/auth.ts

```diff
@@ -281,6 +289,9 @@ if (
       tenantId: env.AUTH_AZURE_AD_TENANT_ID,
       allowDangerousEmailAccountLinking:
         env.AUTH_AZURE_ALLOW_ACCOUNT_LINKING === "true",
+      httpOptions: {
+        agent: proxyAgent,
+      },
```

Comment:
Does it make sense to include this option conditionally, e.g. `...(proxyAgent ? { <content> } : {}`? What happens if the agent is null or undefined here? We  should be sure that it's the default behaviour.

Reply:
It's fair to change. I originally planned to do this but then switched to trying behavior across a couple of AuthProviders. It's safer to exclude it, as you suggested.